tests/integration_tests/adapters/okx/conftest.py

```diff
@@ -0,0 +1,43 @@
+# -------------------------------------------------------------------------------------------------
```

Comment:
Lets remove all of the Python level testing, as we're testing quite thoroughly in Rust now and the adapter will also be used from Rust only as well.

Reply:
removed, thx